pkg/services/object/put/streamer.go

```diff
@@ -175,6 +175,8 @@ func (p *Streamer) preparePrm(prm *PutInitPrm) error {
 		)
 	}
 
+	prm.traverseOpts = append(prm.traverseOpts, placement.WithCopiesNumber(prm.copiesNumber))
```

Comment:
`placement.SuccessAfter` does not fit?

Reply:
`SuccessAfter` does some strange things and is used for read operations. it makes a placement a vector of nodes (not a vector of vectors, so just a concatenation) and says that N number is enough (the first N). so if we have `REP 3` (and some more `REP`s) for the first vector with 10 length and call `SuccessAfter(10)` it will place 7 useless replicas that should be dropped later

Reply:
this is an interesting topic at the intersection of different dimensions

let's try to base the behavior on writing and reading on the same basis. I expect both ops to iterate over same sequence of nodes (unless reading actually ends after the 1st success)

Reply:
https://github.com/nspcc-dev/neofs-node/issues/2616